modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,219 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
 ======
 =====
 
+=== Access Neo4j using a Kubernetes Ingress
+
+The Neo4j Helm charts provide a helm chart that allows you to use a Kubernetes Ingress to access Neo4j running on port _:80_ or _:443_.
```

Comment:
Neo4j is not running on port 80 or 443 ...so we should say access Neo4j on port 80 or 443